src/tools/pontoon/adding_new_project.md

```diff
@@ -34,8 +34,11 @@ Localization for the <a href="https://testpilot.firefox.com/experiments/min-vid"
 * Priority: select priority level from one of the 5 levels available (Lowest, Low, Normal, High, Highest).
 * Contact: select the L10n driver in charge of the project, probably yourself.
 * External Resources: provide links to external resources like l10n preview environment. You need to enter the name and the URL for each resource. You can also pick one of the predefined names: Development site, Production site, Development build, Production build, Screenshots, Language pack.
+* Visibility: determines if the project is visible to the selected users. Pontoon supports the following visibility types:
+  * private (default) - only administrators can access the project.
+  * public - the project is visible for everyone.
 
-Click **SAVE PROJECT** at the bottom of the page, then click **SYNC** to run a test sync. In the [Sync log](https://mozilla-pontoon-staging.herokuapp.com/sync/log/) you should be able to see if it succeeded or failed.
+Click **SAVE PROJECT** at the bottom of the page, then click **SYNC** to run a test sync. In the [Sync log](https://mozilla-pontoon-staging.herokuapp.com/sync/log/) you should be able to see if it succeeded or failed. When everything is synchronized correctly, you can go to the project's admin page and change its visibility to `public`.
```

Comment:
This doesn't seem right: is it really a requirement to wait for a sync? I think you can set the visibility when you create the project, and it would be confusing to set a project as public, sync, then make it disappear.

Reply:
Yeah, you're probably right. I assumed that an administrator has to check the result of the synchronization before the project is shared with translators. If that isn't an obligatory step, I can remove that sentence :-)

Reply:
You can (should!) check the sync while the project is private ;-)

Let's revert to the previous version.

Reply:
done :)